hls4ml/utils/__init__.py

```diff
@@ -1,3 +1,8 @@
 from hls4ml.utils.config import config_from_keras_model, config_from_onnx_model, config_from_pytorch_model  # noqa: F401
 from hls4ml.utils.example_models import fetch_example_list, fetch_example_model  # noqa: F401
 from hls4ml.utils.plot import plot_model  # noqa: F401
+
+try:
```

Comment:
I wonder if there's a better way to do this because now if the import fails, you just gen an error while using it of:
```
AttributeError: module 'hls4ml.utils' has no attribute 'edit_model_configuration'
```
ideally the error message would be better.